### DISCUSSION TITLE
alits开启CDN之后登陆账号报错

### DISCUSSION THREAD

**Discussion by lc99520** [AlistGo/alist#4481] [Q&A]
提示：Guest user is disabled, login please
内网反代端口并开启SSL

**Comment by OneSoftHaha** [comment]
我也是欸！套上CDN后登录提示Guest user is disabled, login please，时不时还会有502（

**Reply by stzbak** [reply]
我也有这个问题，3.29.1版本，直接访问和套cf都没问题。套AWS的cloudfront ，登录提示Guest没开启，开启Guest后可以登录，但无法进入后台。另外，套cloudfront以后Alist-proxy 也无法正常工作，下载的文件内容只有401…

**Reply by stzbak** [reply]
解决了
![微信图片_20231129132507](https://github.com/alist-org/alist/assets/107470278/88aaed7b-aa5e-486c-a809-1d3f69fe022d)
![微信图片_20231129132529](https://github.com/alist-org/alist/assets/107470278/e90fdd02-be23-4b00-9622-f4e7883139f0)

**Comment by jiuaiwo** [comment]
我没呢， 也反戴了 也开了ssl
